phoenix/core/models.py

```diff
@@ -115,6 +118,18 @@ def sales_affected_choice_human(self):
             if s[0] == self.sales_affected_choice
         ][0]
 
+    @property
+    def b2b_partner_has_been_affected(self):
+        return self.b2b_partners_affected_choice == self.YES
+
+    @property
+    def b2b_partner_affected_choice_human(self):
+        return [
```

Comment:
Couldn't you do just `return dict(self.SALES_AFFECTED_CHOICES).get(self.partners_affected_choice)`?

Reply:
Again, I just copied and pasted from the sales part, but I guess you are right, it could work like that. But not sure if I want to change it, I don't trust it that much.